desktop/core/src/desktop/views.py

```diff
@@ -83,12 +84,13 @@
 def is_alive(request):
   return HttpResponse('')
 
+
 def samlgroup_check(request):
   if 'SAML2Backend' in desktop.auth.forms.get_backend_names():
     if REQUIRED_GROUPS.get():
       try:
         userprofile = get_profile(request.user)
-      except:
+      except Exception as e:
```

Comment:
Nit: `except Exception:`

Reply:
changed `except Exception as e`: to `except Exception`